### ISSUE TITLE
added link from vim-ipython readme

### ISSUE THREAD

**Issue by ivanov** [johndgiese/vipy#1]
Hey @johndgiese 
Found your `vipy` project and added it to the list of alternatives to [vim-ipython](https://github.com/ivanov/vim-ipython). Glad you were able to leverage that code to get yourself bootstrapped and up and running. Let me know if you'd prefer to be removed from the [similar projects](https://github.com/ivanov/vim-ipython#similar-projects) portion of the README.

**Comment by johndgiese** [comment]
Thanks Paul,

I am happy to be on your similar projects list!  I had a lot of fun hacking at the code; it taught me a lot about vim and python.  I hope to do a full rewrite this summer, now that I have been using my tool for a few months; I really love ipython and vim and I think the two can make a great pair.  I noticed a while back that you received a grant to work on ipython (if I remember correctly).  Thats great!  How is it going?  Did you attend pycon this year?  Best,

David

**Comment by ivanov** [comment]
Hi David,
I actually learned a fair bit about vim while writing vim-ipython myself, and still learning. Yes, indeed, I'm on the IPython grant from the Sloan foundation, so as soon as I finish my thesis (shotting for May), I'll be on IPython full time for the following year and a half to two years. I did go to pycon - it was a ton of fun, did you go? I see that you're based in Boston - and I was just there, a few weeks ago for the SIAM CSE conference -- too bad we didn't connect before that, wouldn't have been fun to meet up and hack on our vim and ipython projects a bit to figure stuff out together. I'm not sure if you're going to attend SciPy, but maybe we can catch up there - or if you find yourself in the Bay Area - a good chunk of the IPython team all sit together on campus.
